check_logs.py

```diff
@@ -22,7 +21,7 @@
     from ghapi.all import GhApi
     from ghapi.page import paged as gh_paged
 except ModuleNotFoundError:
-    logging.warning("no ghapi library")
+    pass
```

Comment:
Was this change intentional?

Reply:
yes - I got tired of seeing this message - I originally thought it would be helpful, but it isn't